frontend/src/router.js

```diff
@@ -336,6 +336,18 @@ function categoryFromRoute (route) {
   }
 }
 
+function getContentLayout (route) {
+  switch (route.name) {
+    case 'camp/period': return 'full'
+    case 'camp/admin': return 'wide'
+    default: return 'normal'
```

Comment:
Hier gibt es 3 unterschiedliche Return-Werte. In Camp.vue sind aber nur 2 implementiert ('normal' und 'wide').

Reply:
Bei 'full' braucht es keine Beschränkung der Container Breite.